shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -283,19 +413,6 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:kSetEditingStateMethod]) {
     NSDictionary* state = call.arguments;
     [self setEditingState:state];
-
```

Comment:
Removed these because typing too quickly may cause an endless loop between the framework and the text input plugin.

Reply:
I've run into this infinite loop before.  I think the race condition almost never actually happens in reality, so really I think it's a net win to remove this code.  It is concerning that we have no real way to handle the race condition though.